openapi/hub-spid-login.yaml

```diff
@@ -55,6 +57,8 @@ components:
       properties:
         active:
           type: boolean
+      enum:
+        - false

```

Comment:
this two lines have to be indented to the right by two spaces

Reply:
Fixxed